client/nginx.go

```diff
@@ -184,8 +186,8 @@ type Responses struct {
 // Sessions represents stream session related stats.
 type Sessions struct {
 	Sessions2xx uint64 `json:"2xx"`
-	Sessions4xx uint64 `josn:"4xx"`
-	Sessions5xx uint64 `josn:"5xx"`
+	Sessions4xx uint64 `json:"4xx"`
```

Comment:
Could you double check if those typos caused any incorrect behavior?  If that is the case, it's better to fix it in a separate PR labeled as a bug, so that we can properly refer to it in the changelog of the next release. Also, in case of a bug, we will have to update the Prometheus exporter https://github.com/nginxinc/nginx-prometheus-exporter

Reply:
Bug confirmed. Removing from PR.